CIScripts/Test/Tests/Protocol/TunnellingWithAgent.Tests.ps1

```diff
@@ -371,37 +389,48 @@ Describe "Tunnelling with Agent tests" {
     Context "IP fragmentation" {
         # TODO: Enable this test once fragmentation is properly implemented in vRouter
         It "ICMP - Ping with big buffer succeeds" -Pending {
-            Test-Ping `
-                -Session $Sessions[0] `
-                -SrcContainerName $Container1ID `
-                -DstContainerName $Container2ID `
-                -DstContainerIP $Container2NetInfo.IPAddress `
-                -BufferSize 1473 | Should Be 0
-
-            Test-Ping `
-                -Session $Sessions[1] `
-                -SrcContainerName $Container2ID `
-                -DstContainerName $Container1ID `
-                -DstContainerIP $Container1NetInfo.IPAddress `
-                -BufferSize 1473 | Should Be 0
+            $Container1MaxBufferSize = Get-MaxICMPDataSizeForMTU -MTU $Container1NetInfo.MtuSize
+            $Container2MaxBufferSize = Get-MaxICMPDataSizeForMTU -MTU $Container2NetInfo.MtuSize
+
+            $SrcContainers = @($Container1ID, $Container2ID)
+            $DstContainers = @($Container2ID, $Container1ID)
+            $DstIPs = @($Container2NetInfo.IPAddress, $Container1NetInfo.IPAddress)
+            $BufferSizes = @($Container1MaxBufferSize, $Container2MaxBufferSize)
+
+            foreach ($ContainerIdx in @(0, 1)) {
+                $BufferSizeLargerBefore = $BufferSizes[$ContainerIdx] + 1
+                $BufferSizeLargerAfter = $BufferSizes[$ContainerIdx] - 1
```

Comment:
What does `Before` and `After` mean? Maybe `LargerThanMTU`, `SmallerThanMTU` or whatever would be better?

Reply:
It's always larger than MTU, but `Before` means that it is larger before adding tunneling headers and `After` means that it is smaller before and larger after adding the headers. We can rename it to `BufferSizeLargerBeforeTunnellingHeaders` but in my opinion it is to long. What is your opinion? Should we change it to the long version?